libs/dh/wholesale/feature-request-calculation/src/lib/dh-wholesale-request-calculation.html

```diff
@@ -40,11 +43,20 @@
 
         <watt-dropdown
           [label]="t('gridAreas')"
-          [formControl]="form.controls.gridarea"
+          [formControl]="form.controls.gridArea"
           sortDirection="asc"
           [options]="gridAreaOptions$ | push"
         />
 
+        @if (isWholesaleRequest()) {
+        <watt-dropdown
+          translate="wholesale.requestCalculation.resolutionOptions"
+          [label]="t('resolution')"
+          [formControl]="form.controls.resolution"
+          [options]="resolutionOptions"
+          dhDropdownTranslator
+        />
+        } @else {
```

Comment:
nitpick: funny that Prettier does not format this correctly. Hopefully in the future :)